docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md

```diff
@@ -19,6 +19,11 @@ To deploy your application to run continuously or to place it behind a load bala
 
 If you're not familiar with ECS, you can get examples for those files from [here](../../../../examples/#ecs-applications).
 
+## Easier configuration by `pipectl init`
+
+You can generate an application config file easily and interactively by [`pipectl init`](../../command-line-tool.md#generating-an-application-config-apppipecdyaml).
+
+
```

Comment:
```suggestion
Note:

You can generate an application config file quickly and interactively by [`pipectl init`](../../command-line-tool.md#generating-an-application-config-apppipecdyaml).


```

This should be a note, not a new session. wdyt?

Reply:
@khanhtc1202 
Thanks, I fixed all